dll/cpl/desk/lang/tr-TR.rc

```diff
@@ -28,7 +35,7 @@ BEGIN
     PUSHBUTTON "&Renk...", IDC_COLOR_BUTTON, 188, 167, 55, 15
     LTEXT "&Yerleşim:", IDC_STATIC, 188, 128, 55, 9
     COMBOBOX IDC_PLACEMENT_COMBO, 188, 139, 55, 54, CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
-    PUSHBUTTON "Customize &Desktop...", IDC_DESKTOP_CUSTOM, 8, 194, 90, 15
+    PUSHBUTTON "Masaüstünü &Kişiselleştir...", IDC_DESKTOP_CUSTOM, 8, 194, 90, 15
```

Comment:
Again: Please add the correct accelerator at line 33, remove the comment, and don't use &K at line 33. Use what your turkish WIndows 2003 or turkish WinXP does use there as the accelerator!